src/queue.ts

```diff
@@ -75,7 +81,7 @@ export class Queue {
             address
             name
             dao { id }
-            canDelegateCall
+            canDeltegateCall
```

Comment:
this is mispelled, on purpose?

Reply:
not on purpose... not sure why it does not error, though...